app/src/main/java/mozilla/lockbox/presenter/RoutePresenter.kt

```diff
@@ -77,54 +64,18 @@ class RoutePresenter(
     override fun onResume() {
         super.onResume()
         activity.addOnBackPressedCallback(backListener)
-        routeStore.routes
-            .observeOn(mainThread())
-            .subscribe(this::route)
-            .addTo(compositeDisposable)
+//        routeStore.routes
```

Comment:
is there a reason this commented-out code is still here?

Reply:
whoops!